### PR TITLE
fix embargo specs

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made?

fixed the h2 embargo spec so it should pass when DSA is correct.

### CODE REVIEW

File: spec/features/embargo_h2_spec.rb [hunk 1]
```diff
@@ -12,10 +12,10 @@
 RSpec.describe 'H2 item created with embargo; change embargo in Argo',
                type: :feature, stage_only: true, embargo: true, skip_for_default: true do
   let(:collection_title) { RandomWord.nouns.next }
-  let(:item_title) { "Text file for #{collection_title}" }
```
Comment (ndushay) [inline spec/features/embargo_h2_spec.rb:15]: whoops - not unique.  hahaha

File: spec/features/embargo_h2_spec.rb [hunk 2]
```diff
@@ -12,10 +12,10 @@
...
+  let(:item_title) { "Text file #{RandomWord.nouns.next} in H2 collection" }
   let(:user_email) { "#{AuthenticationHelpers.username}@stanford.edu" }
-  let(:purl_xml_embargo_prefix) { '<access type="read">\s*<machine>\s*<embargoReleaseDate>' }
-  let(:purl_xml_embargo_suffix) { '</embargoReleaseDate>\s*<none.*>\s*</machine>\s*</access>' }
+  let(:purl_xml_embargo_prefix) { '<access type="read">\s*<machine>\s*<none.*>\s*<embargoReleaseDate>' }
+  let(:purl_xml_embargo_suffix) { '</embargoReleaseDate>\s*</machine>\s*</access>' }
```
Comment (ndushay) [inline spec/features/embargo_h2_spec.rb:18]: I'm not in love with caring about whether the embargo element is first or last here.   Ideas?   I'm not looking to deal with the entire purl.xml

Comment (jcoyne) [reply spec/features/embargo_h2_spec.rb:18]: xpath query?

Comment (ndushay) [reply spec/features/embargo_h2_spec.rb:18]: so slurp up the entire purl and treat it as xml?   could do.

**Comment by ndushay** [comment]
@jcoyne Now using xpath to find embargo date on purl page and ensure there is only one such node.   This passed on whatever is currently on stage as of right now.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**